README.md

```diff
@@ -104,7 +104,10 @@ J2S returns a result object containing `swagger` and `components` properties. `s
 
 - `joi.alternatives()` defines the structure using the first schema provided on `.items()` (see below for how to override)
 
-- `any.default()` sets the `"default"` detail.
+- `joi.any()`
+  - `.default(5)` -> `"default": 5`
+  - `.example('hi')` -> `"example": "hi"`
+  - `.example('hi').example('hey')` -> `"examples": ["hi", "hey"]`
```

Comment:
You removed the documentation for any.default and replaced it with example code? This doesn't fit with the style of the existing documentation.

Reply:
Ah, I see where you were going. Yeah, for type specific stuff it gets indented below, but `any` isn't a type 😀  So the correct format would be to return `default` back to how it was, and then add a `any.example()` line with a short description.

Reply:
Oooh, gotcha!

Reply:
Perfect, thanks!